### PR TITLE
dockerfile

### PR DESCRIPTION
**Pull request by asabhi6776**
Added dockerfile so that people can build their docker image and run it anywhere.

Thanks

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM php:7.4.27-fpm-buster
+MAINTAINER "asabhi6776"
```
Comment (helich0pper) [inline Dockerfile:2]: LABEL maintainer="asabhi6776"

**Review by helich0pper — CHANGES_REQUESTED** [review]
Thank you for this :) will be useful for beginners

**Comment by asabhi6776** [comment]
Hi, I have updated the Maintainer tag now you can approve the pull request.

**Review by helich0pper — APPROVED** [review]
Thanks

**Pull request merged**